ivy/utils/assertions.py

```diff
@@ -354,17 +354,34 @@ def check_kernel_padding_size(kernel_size, padding_size):
             )
 
 
+def check_inplace_update_support(ensure_in_backend, x):
```

Comment:
let's also make this function private and move it to `exceptions.py` below `handle_inplace_mode`

Reply:
this function is used by the inplace_update funtions in tensorflow and jax backend so I can't make it private.
Unless you want me to inline it where I use it ?